src/ds/hashes.md

```diff
@@ -0,0 +1,47 @@
+Lists, sets, and sorted sets are great for many use cases, but the hash data type is on a higher level. Redis hashes are maps (sequences of field-value pairs) and are used to represent objects in Redis. Consider the following example that shows how to create a hash key using the `HSET` command.
+
+```redis Create a hash
+HSET bike:1 model Deimos brand Ergonom type "Enduro bikes" price 4972
+```
+
+`HSET` returns the number of added key/value pairs.
+
+To retrieve the stored data, use the `HGETALL` command.
+
+```redis HGETALL usage
+HGETALL bike:1 // returns all the field-value pairs associated with the key
+```
+
+If you only want the values of a subset of the fields, use the `HGET` command.
+
+```redis HGET usage
+HGET bike:1 price
+```
+
+You can update fields in a hash using `HSET` by specifying a subset of its field-value pairs.
```

Comment:
- Instead of "update", I think it is worth noting that HSET can be used to add new fields and overwrite existing field values.
- Instead of "field-value pairs" please use "name-value pairs" or "fields". We sometimes use "field" to refer to the pair and sometimes to the name, which is quite misleading.

Reply:
Given your previous comment (the quote from redis.io), it seems I should use field-value. But in this comment you're saying I should use name-value. Which one is the correct definition?

Reply:
A hash field is a name-value pair. Otherwise, the term 'field' is ambiguous.